boards/st/nucleo_u385rg/board.cmake

```
@@ -0,0 +1,11 @@
+# keep first
+board_runner_args(stm32cubeprogrammer "--port=swd" "--reset-mode=hw")
+
+board_runner_args(pyocd "--target=stm32u385rgtxq")
+
+board_runner_args(jlink "--device=STM32U385RG" "--reset-after-load")
```

Unnecessary newlines?
```suggestion
board_runner_args(stm32cubeprogrammer "--port=swd" "--reset-mode=hw")
board_runner_args(pyocd "--target=stm32u385rgtxq")
board_runner_args(jlink "--device=STM32U385RG" "--reset-after-load")
```